cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -525,6 +538,8 @@ class self_comparator {
    * `F(i,j)` returns true if and only if row `i` compares lexicographically less than row `j`.
    *
    * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
```

Comment:
Out of scope for this particular PR, but very much in scope for the doxygen updates overall since I'm sure this tparam is missing docs somewhere: is there any way to define a "standard" docstring for the Nullate parameter (and other similar cases where there's an identical parameter everywhere in the codebase)? It would be nice not to have to write this out (and potentially have slightly divergent versions of this) everywhere in the code base. At least the actual param docstring will be different in different places since we need to document what object contains or doesn't contain nulls, but the template parameter's docstring should be literally identical AFAIK.